docs/lifecycle/README.md

```diff
@@ -0,0 +1,50 @@
+# Set up Object Lifecycle Configuration [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/)
+
+Using object lifecycle configuration on buckets to setup automaic objects deletion after a specified days or a specified date.
+
+## 1. Prerequisites
+- Install MinIO - [MinIO Quickstart Guide](https://docs.min.io/docs/minio-quickstart-guide).
+- Install AWS Cli - [Installing the AWS CLI - AWS Command Line Interface](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+
+
+## 2. Setup a bucket lifecycle using AWS
+
+1. Create a JSON file which contains the bucket lifecycle configuration, as the following example:
+
+```sh
+$ cat >bucket-lifecycle.json
+cat bucket-lifecycle.json
+{
+    "Rules": [
+        {
+            "Expiration": {
+                "Date": "2020-01-01T00:00:00.000Z"
+            },
+            "ID": "Delete very old messenger pictures",
+            "Filter": {
+                "Prefix": "uploads/2015/"
+            },
+            "Status": "Enabled"
+        },
+        {
+            "Expiration": {
+                "Days": "7"
+            },
+            "ID": "Delete temporary uploads",
+            "Filter": {
+                "Prefix": "temporary-uploads/"
+            },
+            "Status": "Enabled"
+        }
+    ]
+}
+```
+
+2. Upload the bucket lifecycle configuration using AWS cli:
+
+```sh
+AWS_ACCESS_KEY_ID="your-access-key" AWS_SECRET_ACCESS_KEY="your-secret-key" aws s3api put-bucket-lifecycle-configuration --bucket your-bucket --endpoint-url http://minio-server-address:port --lifecycle-configuration file://bucket-lifecycle.json
```

Comment:
Would a multi-line instruction, where we set the access key and secret key using `EXPORT ...` shell commands, be easier to read and follow?

Reply:
> Would a multi-line instruction, where we set the access key and secret key using `EXPORT ...` shell commands, be easier to read and follow?

fixed